design/README.md

```diff
@@ -0,0 +1,32 @@
+# Architecture Decision Records Folder
+This folder contains EdgeX Foundry decision records (ADR) and legacy design / requirement documents.
+
+    /design
+        /adr (architecture decision Records)
+        /legacy-design (legacy design documents)
+        /legacy-requirements (legacy requirement documents)
+
+At the root of the ADR folder (/design/adr) are decisions that are relevant to multiple parts of the project (aka � *cross cutting concerns*).  Sub folders under the ADR folder contain decisions relevant to the specific area of the project and essentially set up along working group lines (security, core, application, etc.).
+
+
+## Naming and Formatting
+ADR documents are requested to follow RFC (request for comments) naming standard.  Specifically, authors should name their documents with a sequentially increasing integer (or serial number) and then the architectural design topic:  (sequence number - topic).  Example:  0001-SeparateConfigurationInterface.  The sequence is a global sequence for all EdgeX ADR.  
```

Comment:
FYI, Mike and I are good with `sequence is a global sequence for all EdgeX ADR` as long as we enforce the TOC is updated as part of the PR for a new ADR.

Reply:
Yep - as stated in the last line of the Readme:  Document authors are asked to keep the TOC updated with each new document entry.